lib/genome/importers/foundation_one_genes/foundation_one_genes.rb

```diff
@@ -0,0 +1,46 @@
+require 'genome/online_updater'
+
+module Genome; module Importers; module FoundationOneGenes;
+class FoundationOneGenes < Genome::OnlineUpdater
+  attr_reader :file_path, :source
+
+  def initialize(file_path)
+    @file_path = file_path
+  end
+
+  def import
+    remove_existing_source
+    create_new_source
+    create_gene_claims
+  end
+
+  private
+  def remove_existing_source
+    Utils::Database.delete_source('Foundation One')
+  end
+
+  def create_new_source
+    @source ||= DataModel::Source.create(
+        {
+            base_url: 'https://www.foundationmedicine.com/portfolio',
+            citation: 'High-throughput detection of actionable genomic alterations in clinical tumor samples by targeted, massively parallel sequencing. Wagle N, Berger MF, ..., Meyerson M, Gabriel SB, Garraway LA. Cancer Discov. 2012 Jan;2(1):82-93',
+            site_url: 'http://www.foundationone.com/',
+            source_db_version: '9-Oct-2013',
+            source_type_id: DataModel::SourceType.INTERACTION,
+            source_trust_level_id: DataModel::SourceTrustLevel.EXPERT_CURATED,
+            source_db_name: 'Foundation One',
```

Comment:
We generally have not included spaces in the source_db_name. If you change it here, it will also need to be changed in the `remove_existing_source` method.

Reply:
I thought this was weird but I just pulled it from the last importer for Foundation One. Just looked in the db and it's listed as `FoundationOneGenes`

Reply:
I think that's what we should use as the source_db_name then

Reply:
This has been changed.